test/components/Charts.test.js

```diff
@@ -109,36 +70,25 @@ describe('<Charts />', () => {
                 startDate : moment().subtract(1, "days").toDate(),
                 endDate : new Date(Date.now())
             },
-            dataSets : new Map()
-    }
-
-        function createRandomData() {
-            const oneHour = 3600000;
-            let data = {label: '', values: []};
-            let startDate = new Date(2016, 5, 5);
-            let startMillis = startDate.getTime();
-            for (let i = 0; i < 60; i++) {
-                let multi = Math.floor(Math.random() * 5);
-                startMillis += oneHour * multi;
-                data.values.push({x: new Date(startMillis), y : Math.floor(Math.random() * 10)});
-            }
-            return data;
-        };
+            dataSets : Immutable.Map()
+        }
 
         const wrapper = shallow(<Charts {...testProps}/>);
 
         expect(wrapper.children().length).toBe(0);
     });
 
     it('renders the only matching version data', () => {
+        let dataSets = Immutable.Map();
+        dataSets = dataSets.set('id1', createRandomData());
```

Comment:
I've seen this `createRandomData` function a couple of times now, maybe you can refactor it in some kind of utility module.